pages/index.js

```diff
@@ -1,10 +1,17 @@
-
 export default function Home() {
-  return (
-    <div className='h-screen bg-slate-900 text-9xl font-extrabold flex items-center justify-center text-center text-blue-500 uppercase'>
-      <p>
-        we are almost here...
-      </p>
-    </div>
-  )
+    return (
+        // <div className='flex items-center justify-center h-screen font-extrabold text-center text-blue-500 uppercase bg-slate-900 text-9xl'>
```

Comment:
also the same comment I left about `svgs` would applied for project images instead of being named `project1` or `project2` why don't we name them according to the actual project name from Figma since they are unique. it would be more easier to differentiate images